aibolit/ast_framework/ast.py

```diff
@@ -80,7 +80,7 @@ def __iter__(self) -> Iterator[ASTNode]:
         for node_index in self.tree.nodes:
             yield ASTNode(self.tree, node_index)
 
-    def get_subtrees(self, root_type: ASTNodeType) -> Iterator['AST']:
+    def get_subtrees(self, *root_type: ASTNodeType) -> Iterator['AST']:
```

Comment:
Does `root_type` has list type?

Reply:
Yes, this is mypy syntax. Looks strange. You specify just single item type, but get list if them.